### DISCUSSION TITLE
ISSUE - Scenario won't be destroyed

### DISCUSSION THREAD

**Discussion by fuzzymango** [SadServers/sadservers#61] [Feedback and Ideas]
Hey all,
I'm fairly new to sadservers and I think I've ran into an issue. I accidentally started a scenario that I already solved so I wanted to kill that server and try another problem. The issue is that when I click "destroy" in the "Your Running Servers" page of the dashboard, nothing seems to happen and the server doesn't get destroyed. So now I'm stuck not being able to start any new servers until this dies.
![image](https://github.com/user-attachments/assets/8f030a71-cb2e-4a4d-9215-00af18cada88)

**Comment by fduran** [comment]
Thanks @fuzzymango for reporting this, I've had some people observing the same or similar issue. I'll try and reproduce.

**Comment by fduran** [comment]
Hello @fuzzymango , I fixed a bug so this issue should be solved (unless there's another bug...)
